Turntable/turntable.js

```diff
@@ -303,23 +290,23 @@ window.addEventListener("load", rotateTurntable = () => {
   nextSong.addEventListener("dblclick", handleFadBtn);
 
   fadBtn.addEventListener("click", () => {
-    const topbarContentFadeIn = document.querySelector(".main-entityHeader-topbarContentFadeIn");
     const fullAppDisplay = document.querySelector("#full-app-display");
 
+    fullAppDisplay.appendChild(songPreviewContainer);
+
     if (!clickedFadBtn) {
       if (+localStorage.getItem("enableBlurFad")) fullAppDisplay.dataset.isBlurFad = "true";
-      fullAppDisplay.appendChild(songPreviewContainer);
 
       handleFadControl();
 
       fullAppDisplay.addEventListener("contextmenu", () => handleContextMenu(fullAppDisplay), { once: true });
 
-      fullAppDisplay.addEventListener("dblclick", () => handleMainInterface());
+      // fullAppDisplay.addEventListener("dblclick", () => handleToggleFad());
 
       clickedFadBtn = true;
     }
 
-    handleMainInterface("active", topbarContentFadeIn);
+    // handleToggleFad(true);
```

Comment:
Are both of these commented on purpose? The function never gets called this way 🤔

Reply:
That's legacy code. I want to quickly process invalid code fisrt, and i will change it later.